roles/cases/cases_get_open_cases/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: Retrieve a list of open cases
+  cp4s_get_open_cases:
+  register: open_cases
+
+- name: Print out list of open cases
+  debug:
+    msg:
+      - "open_cases"
+      - "ID: {{ item.id }}"
+      - "Name: {{item.name}}"
+  loop: "{{ open_cases.case }}"
```

Comment:
Any chance of some comments on the playbook. Love the use of looping

Reply:
Yeah will do, ill add them and make the changes